src/apps/components/AppDetailsPage/DataPrivacyCard.tsx

```diff
@@ -1,47 +1,51 @@
-// @ts-strict-ignore
-import CardTitle from "@dashboard/components/CardTitle";
 import ExternalLink from "@dashboard/components/ExternalLink";
 import Skeleton from "@dashboard/components/Skeleton";
-import { Card, CardContent } from "@material-ui/core";
+import { Box, BoxProps, Text } from "@saleor/macaw-ui/next";
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 
 import messages from "./messages";
-import { useStyles } from "./styles";
 
-interface DataPrivacyCardProps {
+type DataPrivacyCardProps = {
   dataPrivacyUrl?: string | null;
   loading: boolean;
-}
+} & BoxProps;
 
-const DataPrivacyCard: React.FC<DataPrivacyCardProps> = ({
+export const DataPrivacyCard: React.FC<DataPrivacyCardProps> = ({
   dataPrivacyUrl,
   loading,
+  ...boxProps
 }) => {
-  const classes = useStyles();
   const intl = useIntl();
 
   if (!dataPrivacyUrl && !loading) {
     return null;
   }
 
+  const renderContent = () => {
+    if (loading) {
+      return <Skeleton />;
+    }
+
+    if (dataPrivacyUrl) {
+      return (
+        <ExternalLink href={dataPrivacyUrl} target="_blank">
+          <FormattedMessage {...messages.dataPrivacyDescription} />
+        </ExternalLink>
+      );
+    } else {
+      return <Text>{intl.formatMessage(messages.noDataPrivacyPage)}</Text>;
+    }
+
+    throw new Error("Leaking switch statement");
```

Comment:
**suggestion**: shouldn't it be logged into sentry?

Reply:
Won't sentry automatically do that? Its unhandled on purpose

Reply:
I added error boundary and explicitely called Sentry